src/path_info.cpp

```diff
@@ -158,6 +158,12 @@ void PATH_INFO::set_standard_filenames(void)
     update_pathname("obsolete-mods", FILENAMES["moddir"] + "obsolete-mods.json");
     update_pathname("defaultsounddir", FILENAMES["datadir"] + "sound");
 
+    // User's data directories
+    update_pathname("user_datadir", FILENAMES["user_dir"] + "data/");
+    update_pathname("user_gfxdir", FILENAMES["user_datadir"] + "gfx/");
```

Comment:
> all you need to do is define that path as FILENAMES ["user_dir"] + "gfx".

I see what you're saying. What's irking me is that the standard gfx, sound, and mods are in the same dir when a package manager would create a package for this (i.e. by setting PREFIX at compile time). I suppose I could mirror that by conditionally compiling based on DATA_DIR_PREFIX here, as done on line 118.

> That's a hilarious question. That is the default, and the dominant way the game is packaged and distributed.

Hmm.. interesting that most people install it that way. I wonder how much that'd change if users had a way to install it via their os's package manager, like in gentoo.